wagtail_storages/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.1"
```

Comment:
According to semver, this should be at least 1.1.0, because it drops support for older Wagtail versions